documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -15,7 +15,7 @@
 
 * npm lockfile and shrinkwrap detectors now ignore packages flagged as extraneous in the package-lock.json and npm-shrinkwrap.json files.
 * Support added for Opam Package Manager via [Opam Detector](packagemgrs/opam.md).
-* (IDETECT-4441) New Gradle Native Inspector option to only process the root dependencies of a Gradle project. See [detect.gradle.root.only](properties/detectors/gradle.md#gradle-root-only-advanced) for more details.
+* (IDETECT-4441) New Gradle Native Inspector option to only process the root dependencies of a Gradle project. See [detect.gradle.root.only](properties/detectors/gradle.md#gradle-root-only-enabled-advanced) for more details.
```

Comment:
Can you remove the ticket number. Those aren't included for new features.

Reply:
Indeed, only for customer facing fixes. Good eye!

Reply:
Done.